benchmarks/CMakeLists.txt

```
@@ -62,4 +62,11 @@ ConfigureBench(RANDOM_ALLOCATIONS_BENCH "${RANDOM_ALLOCATIONS_BENCH_SRC}")
 set(REPLAY_BENCH_SRC
   "${CMAKE_CURRENT_SOURCE_DIR}/replay/replay.cpp")
 
-ConfigureBench(REPLAY_BENCH "${REPLAY_BENCH_SRC}")
\ No newline at end of file
+ConfigureBench(REPLAY_BENCH "${REPLAY_BENCH_SRC}")
+###################################################################################################
+# - uvector benchmark --------------------------------------------------------------------------------
+
+set(UVECTOR_BENCH_SRC
+  "${CMAKE_CURRENT_SOURCE_DIR}/device_uvector/device_uvector_bench.cu")
+
+ConfigureBench(UVECTOR_BENCH "${UVECTOR_BENCH_SRC}")
```

Is it necessary to have new line at EOF here?

---

Yes, every file should end with a newline. This is important for unix utilities and the C preprocessor -- basically anytime you might concatenate files. So it's best to just do it in all files.  But it's easy to miss, which is why it's nice that Github flags it.

I always just add the suggestion so the author can just click a button to accept it.

```suggestion
ConfigureBench(UVECTOR_BENCH "${UVECTOR_BENCH_SRC}")

```

---

FYI: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file

---

> so the author can just click a button to accept it.

afaik, each time you merge a suggestion, it kicks off CI :disappointed:

---

Click the "add suggestion to batch" button.
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request

---

How have I never seen that before....